modules/test/playwright/tests/osb-faro-web/utils/utils.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, expect} from '@playwright/test';
+
+export async function createIndividuals(apiHelpers, names: string[]) {
```

Comment:
@ViniLopes20 maybe we can have a separated file for individuals functions, since we probably will have more utility function for this context